web/thesauruses/_meta/rules.json

```diff
@@ -0,0 +1,13 @@
+{
+  "meta": {
+    "structure": "rules",
+    "structure_name": "Rules"
+  },
+  "categories": {
+    "Defining Rules": {
+	"defrule": "Defining Rules"
```

Comment:
For these, I'd rather spell out things and add underscores. So like
`defrule` 
becomes
`defining_rules`
It makes it a bit easier to understand for people that are glancing at them in language files (since they won't necessarily have the written out forms yet).